app/components/ui-kit/box.js

```diff
@@ -95,13 +95,14 @@ const POSITION_INSETS = {
 
 // Height & Width
 const MAX_WIDTHS = {
-  XXS: '2xs',
+  XS2: '2xs',
   XS: 'xs',
   SM: 'sm',
   MD: 'md',
   LG: 'lg',
   XL: 'xl',
-  XXL: '2xl',
+  XL2: '2xl',
+  XL6: '6xl',
```

Comment:
These keys aren't referenced anywhere else yet, so these should be fine to change. I think this convention is more practical, and it also matches the convention used for sizes in the text component.